tests/Integrations/Elasticsearch/V1/ElasticSearchSandboxedIntegrationTest.php

```diff
@@ -0,0 +1,562 @@
+<?php
+
+namespace DDTrace\Tests\Integrations\Elasticsearch\V1;
+
+use DDTrace\Integrations\ElasticSearch\V1\ElasticSearchIntegration;
+use DDTrace\Tests\Common\IntegrationTestCase;
+use DDTrace\Tests\Common\SpanAssertion;
+use Elasticsearch\Client;
+
+/**
+ * Tests for Elasticsearch Client. We test specifically only most commonly used tests, for the other tests we just make
+ * sure that if a non existing method is provided, that for example does not exists for the used client version
+ * the integration does not throw an exception.
+ */
+class ElasticSearchSandboxedIntegrationTest extends IntegrationTestCase
```

Comment:
I know this is a preview PR, so I'm just assuming you're going to eventually extend this from `ElasticSearchIntegrationTest`, right? :)

Reply:
Actually here the rule `IS_SANDBOX=true|false` does not apply as for most of the tests are parent child so the order is the same for sandboxed and not sandboxed.